### PR TITLE
bump!: Release 2.0.0 - Decidim dependency to v0.27

### PR DESCRIPTION
**Pull request by Quentinchampenois**
#### Description

Bump module to work with Decidim 0.27

#### Note
🚨 **Breaking change**
Between version `0.26` and `0.27` file upload has changed to Direct Upload. Module now supports direct upload

#### How to test 

Install dev app :
* `bundle install`
* `bundle exec rake development_app`
* `cd development_app && bundle exec rails s`

* Log in as admin
* Go to BO
* Navigates to budgets in a participatory process
* In budget navigates to related projects
* In projects index, Export all projects
* Then select `Import projects to budget` in the `import` dropdown
* Upload the previously exported file
It should work and you should be redirected to the projects index

**Try to generated errors in file, you can remove required params, link to bad ids etc**

#### Tasks

- [x] Use OpenSourcePolitics Github Actions in CI
- [x] Add Release and Publish Actions

### CODE REVIEW

File: lib/decidim/budgets_importer/engine.rb [hunk 1]
```diff
@@ -18,6 +18,10 @@ class Engine < ::Rails::Engine
       initializer "BudgetsImporter.webpacker.assets_path" do
         Decidim.register_assets_path File.expand_path("app/packs", root)
       end
+
```
Comment (eliegaboriau) [inline lib/decidim/budgets_importer/engine.rb]: Is that supposed to be in this module ?

File: yarn.lock [hunk 2]
```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (eliegaboriau) [inline yarn.lock]: Is this file useful ?

**Review by eliegaboriau — APPROVED** [review]
LGTM, juste a few interrogations about two files

File: lib/decidim/budgets_importer/engine.rb (hunk 1 above)
Comment (Quentinchampenois) [reply lib/decidim/budgets_importer/engine.rb]: I think so, it registers the `packs` directory in the Decidim registered paths registry

File: yarn.lock (hunk 2 above)
Comment (Quentinchampenois) [reply yarn.lock]: Nice catch I will remove this one

**Review by armandfardeau — APPROVED** [review]

**Review by Ouardaoubahri — APPROVED** [review]
LGTM ! Thank you 🙌

**Pull request merged**